Source/Core/DolphinQt2/DolphinQt2.manifest

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I think it's probably better to be self-contained as we can't make references to files bundled with Visual Studio with confidence (like when cross compiling).
We can certainly update this manifest to have the per monitor DPI aware setting.